### PR TITLE
Create CR - remove :project & :email

### PR DESCRIPTION
**Pull request by stejskalleos**
:project & :email are loaded from the json file

**Tested commands**
Create CR:
```
hammer compute-resource create --name gce_ham \
--provider GCE --key-path ~/path/to/file.json --zone europe-west1-b
```
Create VM:
```
hammer host create \
--name "myhost$EPOCHSECONDS" \
--organization "Default Organization" \
--location "Default Location" \
--compute-resource "gce_ham" \
--provision-method 'image' \
--root-password "changeme" \
--interface "type=interface,domain_id=1,managed=true,primary=true,provision=true" \
--architecture x86_64 \
--operatingsystem "CentOS_Stream 8" \
--compute-attributes "machine_type=e2-micro,associate_external_ip=true" \
--volume="size_gb=23" \
--image "stream8"

```

### CODE REVIEW

File: lib/hammer_cli_foreman_google/compute_resources/google.rb [hunk 1]
```diff
@@ -25,8 +25,6 @@ def volume_attributes
 
       def provider_specific_fields
         [
-          Fields::Field.new(label: _('Project'), path: [:project]),
-          Fields::Field.new(label: _('Email'), path: [:email]),
```
Comment (ofedoren) [inline lib/hammer_cli_foreman_google/compute_resources/google.rb:29]: Those are not options, but rather fields to display for list/info commands. If we receive this information from the server then we shoudn't remove it.

Comment (stejskalleos) [reply lib/hammer_cli_foreman_google/compute_resources/google.rb:29]: But we don't get these anyway, we removed the fields in plugin:

```bash
curl http://foreman.com/3000/api/compute_resources/6
{
  "description": null,
  "url": "europe-west1-a",
  "created_at": "2022-10-31 07:48:26 UTC",
  "updated_at": "2022-10-31 07:48:26 UTC",
  "id": 6,
  "name": "gce_ham2",
  "provider": "GCE",
  "provider_friendly_name": "Google",
  "zone": "europe-west1-a",
  "images": [],
  "compute_attributes": [],
  "locations": [
    {
      "id": 4,
      "name": "Default Location",
      "title": "Default Location",
      "description": null
    }
  ],
  "organizations": [
    {
      "id": 3,
      "name": "Default Organization",
      "title": "Default Organization",
      "description": null
    }
  ]
}
```

**Comment by adamruzicka** [comment]
I can't create a compute resource using hammer
```
# hammer compute-resource create --name gce_ham --provider GCE --key-path ~/test.json --zone europe-west1-b
Could not create the compute resource:
  incorrect/invalid --provider option
```

compute-resource info shows nothing
```
# hammer compute-resource list
---|----------------------|---------
ID | NAME                 | PROVIDER
---|----------------------|---------
5  | gogol                | Google  
6  | gogol coconut-island | Google  
---|----------------------|---------

# hammer compute-resource info --id 6; echo stop
stop
```

**Comment by stejskalleos** [comment]
Tested commands:
```
hammer compute-resource create
hammer compute-resource list
hammer compute-resource info
hammer host create
```
all worked fine, do you have correct branches everywhere?

* foreman: ls/remove_gce (https://github.com/theforeman/foreman/pull/9272)
* foreman_google: ls/replace_core (https://github.com/theforeman/foreman_google/pull/54)
* hammer-cli-foreman-google: ls/cr_create (this PR)

**Review by adamruzicka — APPROVED** [review]
Mystery solved, I had all the right patches, but have not enabled the hammer plugin in configuration. Seeing things coming from hammer-foreman-google in --help made me believe everything was right. Sorry about the noise, all is well

**Comment by stejskalleos** [comment]
Cool, merging the PR then, thanks @adamruzicka @ofedoren !

**Pull request merged**